### DISCUSSION TITLE
CI-Tests Possibilities

### DISCUSSION THREAD

**Discussion by agarshiv** [ossf/scorecard#4154] [Q&A]
Currently in CI-Test we are checking check runs and status for the commits.
It's possible to get more information about specific check runs for example we want to check if one of the check runs are really running unit tests. It will be good if we can get access to logs and can find some specific keywords or schemas often included inside the unit test.
Maybe somebody add similar functionality or if not, this functionality can be added in scorecard?

**Comment by agarshiv** [comment]
Hi Scorecard,
Any updates regarding Ci-Tests Criterion for this ticket.

**Comment by spencerschrock** [comment]
I don't believe this exists currently, and the closest existing request for this is probably #376. I think one issue with this is the hundreds of ways to run unit tests, so there would be false negatives.

**Reply by CsatariGergely** [reply]
Yes, false negatives can be a problem, but I think we can figure out some logic to handle them. I add some brainstorming to the issue.
